7.txt

```diff
@@ -0,0 +1 @@
+asdf7
```

Comment:
There's a bug here, this should say `defg7`

Reply:
Made a fix, take a look.